services/galley/src/Galley/API/Query.hs

```diff
@@ -398,6 +406,22 @@ conversationIdsPageFromV2 lusr Public.GetMultiTablePageRequest {..} = do
           mtpPagingState = Public.ConversationPagingState table (LBS.toStrict . C.unPagingState <$> pwsState)
         }
 
+    -- MLS self-conversation of this user
+    selfConvId = mlsSelfConvId (tUnqualified lusr)
+    isNotSelfConv = (/= selfConvId) . qUnqualified
```

Comment:
How about the global team conversation?

Reply:
I'd suggest to do that in a separate PR. I suppose it won't be hard to add it on top of this (assuming it gets merged), but you would know best.